### ISSUE TITLE
[MHV SM / Accessibility] Change of context highlight

### ISSUE THREAD

**Issue by steveg-IAT** [department-of-veterans-affairs/va.gov-team#122567]
### 📝 User story

As a user,
I expect that the correct highlight state is set on the message-box links,
so that I know where I am in the application.

### 📝 Problem statement

Initial page load missing trailing /' that triggers highlighting of inbox.  
Once I click the "Inbox" link, the correct highlight / active is correctly applied.

This also happens anytime you are on the SM product and use the "Messages" navigation element in the My HealthyVet level navigation.

### 🌌 Supporting screenshots or videos

<img width="822" height="817" alt="Image" src="https://github.com/user-attachments/assets/26514a0e-c5a7-4a28-ac16-e250fe92a878" />

### 👣 Reproduction steps

On [Staging](https://staging.va.gov/my-health/secure-messages/inbox), Using [Test User #4](https://github.com/department-of-veterans-affairs/va.gov-team-sensitive/blob/master/teams/sm/test-accounts.md)

1. Navigate to MHV SM inbox
2. > Notice that the "Inbox" link component does not have it's corrrect highlight / active state (text should be bold and there should be a 5px blue bottom border.

### 🔗 Point of contact

@steveg-IAT

### ⚠️ Severity level

- 4, Minor. Consider fixing or exploring in 2-4 sprints post-launch.

### 🔎 Relevant standards

[3.2.2(a)](https://www.w3.org/TR/WCAG22/#on-input) Changing control settings triggers a change of context

### 𝌣 Proposed solution or next steps

- Handle both cases of URLs (with and without trailing slash)

### ✅ Acceptance criteria

- [ ] The Inbox tab link will be highlighted whe the user reaches the SM langing page initially.

### ✅ Possible tasks

- [ ] Review, refine, and acknowledge feedback.
- [ ] Update ticket information (title, assignee, labels, project(s), etc.)
- [ ] Fix and/or document decisions made.
- [ ] Product manager to move ticket in their respective product board to 'QA/A11y Review' after reviewing documented decisions / validating fix.
- [ ] Accessibility specialist will close the ticket after reviewing documented decisions / validating the fix.

**Comment by steveg-IAT** [comment]
NOTES:

vets-website/src/platform/mhv/secondary-nav/containers/MhvSecondaryNav.jsx
~ line # 39:
```
{
    title: 'Messages',
    actionName: `${actionPrefix} - Messages`,
    icon: 'forum',
    appRootUrl: '/my-health/secure-messages',
    href: `/my-health/secure-messages/inbox`,
  },
```

    href: `/my-health/secure-messages/inbox`, is missing a trailing slash.

- [x] Fix issue
- [ ] Review and update unit tests

**Comment by steveg-IAT** [comment]
[PR #41549](https://github.com/department-of-veterans-affairs/vets-website/pull/41549) IN
- [x] Tests Past
- [x] Reviewed
- [x] Merged

**Comment by sarahhortonadhoc** [comment]
@steveg-IAT looks good — will move to Done!
